README.md

```diff
@@ -2,72 +2,58 @@
 
 For steps on how to work with this repository [please see here](https://docs.labs.lambdaschool.com/labs-spa-starter/)
 
-🚫 Note: All lines that start with 🚫 are instructions and should be deleted before this is posted to your first Pull-Request is merged. This is intended to be a guideline. Feel free to add your own flare to it.
-
 # Project
 
-You can find the deployed project at [🚫URL NAME GOES HERE](🚫copy and paste URL here).
```

Comment:
Looks like we are live at https://labspt13-resfeber-b-be.herokuapp.com/

Reply:
I had a little confusion about whether or not we were linking the deployed full-product or just the BE, so thank you for clarifying that.